pyproject.toml

```
@@ -0,0 +1,6 @@
+# Ruff configurations
+[tool.ruff]
+extend-exclude = ["data", "models", "db", "cache"]
```

Based on the configurations in the `.flake8` file, we only need to exclude `data`. The `tools` directory is not used, and the rest (i.e., `.git` and `__pycache__`) should be already excluded with the default configs.
```suggestion
extend-exclude = ["data"]
```

---

Done